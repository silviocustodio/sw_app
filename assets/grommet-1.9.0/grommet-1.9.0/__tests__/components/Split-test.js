// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react-test-renderer';

import Split from '../../src/js/components/Split';

// needed because this:
// https://github.com/facebook/jest/issues/1353
jest.mock('react-dom');

describe('Split', () => {
  it('has correct default options', () => {
    const component = renderer.create(
       <Split><div>left</div><div>right</div></Split>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
