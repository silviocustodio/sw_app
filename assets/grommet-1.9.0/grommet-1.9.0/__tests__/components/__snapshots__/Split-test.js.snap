// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Split has correct default options 1`] = `
<div
  className="grommetux-split"
>
  <div
    className="grommetux-split__column grommetux-split__column--fixed grommetux-split__column--flex"
  >
    <div>
      left
    </div>
  </div>
  <div
    className="grommetux-split__column grommetux-split__column--fixed grommetux-split__column--flex"
  >
    <div>
      right
    </div>
  </div>
</div>
`;
